File: Backend/MS_Commande/src/main/java/com/example/demo/Model/CommandeDetail.java
package com.example.demo.Model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
@NoArgsConstructor
@AllArgsConstructor
@Data 

public class CommandeDetail {
	@Id
	@GeneratedValue
	private int id;
	private String Livre;
	private int Qte;

}

File: front/MSprojet/src/app/commande-detail.ts
import { Input } from "@angular/core";

export class CommandeDetail {
    Livre!:string;
    Qte!:Number;
    
    public get livre(){
        return this.Livre;
    }
   
    public get qte(){
        return this.Qte
    }
    public set livre(value: string){
        this.Livre=value;
    }
    public set qte(va: Number){
        this.Qte=va;
    }
}
